examples/jsm/controls/OrbitControls.js

```diff
@@ -614,12 +614,21 @@ class OrbitControls extends EventDispatcher {
 
 		function updateMouseParameters( event ) {
 
-			if ( ! scope.zoomToCursor ) {
+			if ( !scope.zoomToCursor ) {
 
 				return;
 
 			}
 
+			if ( !scope.object.isPerspectiveCamera && !scope.object.isOrthographicCamera ) {
```

Comment:
Do you mind undoing this check? We can add it when people actually complain about it...